### DISCUSSION TITLE
Welcome to Acrypto - Algo trading weighted strategy Discussions!

### DISCUSSION THREAD

**Discussion by AlbertoCuadra** [AlbertoCuadra/algo_trading_weighted_strategy#22] [Announcements]
## 👋 Welcome traders!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Exilewip** [comment]
Hi Alberto,

first, thanks for the work. Very interesting !
IMO you're on the good way.

I was thinking about the possibility to know which of the 5 indicators have been triggered for each trade.
This kind of feature can give us an analytics views and can help to understand / adjust the algorithm setup. Maybe a board like the perf one ?
What do you think ? Is my explaination clear ? (Not sure about my english speaking lol)

Second feature proposition, but not sure it's possible :
Have several indicators (more than the 5 you actually integrated) and be able to activate/desactivate each one in the setup menu. 
The goal is to be able to make your own combination of indicators. Maybe a little library.

Can't wait for the pro and premium version. Want to leverage hard haha

You rocks !
++

**Reply by AlbertoCuadra** [reply]
Hello @Pultre,

Sorry for the late reply, I have been very focused on my research in the last few months. Your first suggestion is quite interesting and should actually help significantly during the tuning optimization process. As for the second, I think it's possible, at least I have an idea how to do it. At the moment you can activate/deactivate any of the included strategies.

I'm not sure when I'll be back on this project, but these features should definitely be included.

Best,
Alberto
